src/components/forms/validations.ts

```diff
@@ -88,18 +88,18 @@ export const mentorSpecificValidation = z.object({
     reasonToBeMentor: z
         .string()
         .min(1, "Please tell us why you want to be a mentor at HawkHacks."),
-    linkedinUrl: z
-        .string()
-        .url("Please input a valid url for your LinkedIn.")
-        .optional(),
-    githubUrl: z
-        .string()
-        .url("Please input a valid url for your GitHub.")
-        .optional(),
-    personalWebsiteUrl: z
-        .string()
-        .url("Please input a valid url for your personal website.")
-        .optional(),
+    // linkedinUrl: z
+    //     .string()
+    //     .url("Please input a valid url for your LinkedIn.")
+    //     .optional(),
+    // githubUrl: z
+    //     .string()
+    //     .url("Please input a valid url for your GitHub.")
+    //     .optional(),
+    // personalWebsiteUrl: z
+    //     .string()
+    //     .url("Please input a valid url for your personal website.")
+    //     .optional(),
```

Comment:
Commented this out because the links are optional and it was getting in the way even with them being `.optional()`...